### DISCUSSION TITLE
UIController init()

### DISCUSSION THREAD

**Discussion by Camputron** [AlmasB/FXGL#1357] [Q&A] [answered]
I have a controller class that implements UIController. I notice I have to implement a method called init(). 

I'm wondering when init is invoked. I've tried printing out some stuff but it's never executed.

**Comment by AlmasB** [comment, marked as answer]
Hi @CamposmDev 

It is executed right after the FXML file is loaded:

```
var controller = ...
UI fxmlUI = getAssetLoader().loadUI("test.fxml", controller);
```
